### PR TITLE
PY2/PY3 compatibility changes + follow pep8 standard

### PR DESCRIPTION
**Pull request by roks0n**
No major changes. Got rid of using different approaches (`if __PY3__ .... else ....`) due to py2/py3 compatibility where possible and used six's and futurez functionalities for that. Also, followed pep8 standard in files where I did changes.

### CODE REVIEW

File: arky/util.py [hunk 1]
```diff
@@ -1,350 +1,375 @@
...
 
 def dumpJson(cnf, name, folder=None):
-	filename = os.path.join(HOME if not folder else folder, name)
-	out = io.open(filename, "w" if __PY3__ else "wb")
-	json.dump(cnf, out, indent=2)
-	out.close()
-	return os.path.basename(filename)
+    filename = os.path.join(HOME if not folder else folder, name)
+    out = io.open(filename, "w")
+    json.dump(cnf, out, indent=2)  # what exactly do we want to do here?
```
Comment (roks0n) [inline arky/util.py]: we just seem to dump JSON, but only returning the path? What is the intended behavior?

**Comment by Moustikitos** [comment]
Hi,
Simplification is always good.
Did you tried untitesting with both python 2.x and 3.x ?
Did you run `cli` to check if all is runing fine ?

File: arky/util.py (hunk 1 above)
Comment (Moustikitos) [reply arky/util.py]: `dumpJson`, is to write a dictionary into a file. I think in `cli` i use returned path once or twice.

**Comment by roks0n** [comment]
- I did run the unit tests, both in py2 and py3. They also passed in jenkins:https://travis-ci.org/ArkEcosystem/arky/builds/339874173
- I'll run cli locally - we need to write tests for that! :)

Not that tabs/spaces debate.  :D In python I always use spaces which is also the preferred way: https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces

**Comment by rodxavier** [comment]
nice work @roks0n. I was thinking if it's possible to replace some `json.load(request.get(...))` to `request.get(...).json()`.

**Comment by roks0n** [comment]
Ty @rodxavier. Definetelly, I think we should change that.

**Comment by Moustikitos** [comment]
I'm okay with `requests.get(...).json()`... it is cleaner (i use it on my failover work)
The current version is working fine with cli till now (i use it daily running it from source)

Did someone test the `ldgr.py` modifications with a real ledger wallet ? it is the only "dark" point in this commit (see the comment).

We should consider also unittesting our work on ubuntu.

**Comment by roks0n** [comment]
- What part broke for you in CLI? I'll check later on today when I get back on my computer.
- I'll check if there's any library that mocks ledger (or anything else) that we could use in our tests.
- Travis is already running all of our tests on Ubuntu (trusty).

**Comment by roks0n** [comment]
Oh bloody hell, I ordered a Ledger wallet but because it's out of stock they ship in next on 27th of March.

**Comment by Moustikitos** [comment]
The CLI is not broken, I meant that till now, the cli is still in good shape even with no unittests...
here is a test about the modif done in `ldgr.py`:
```
C:\Users\Bruno>py -2.7
Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 20:40:30) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import struct
>>> struct.pack(">I", 0x80000000)
'\x80\x00\x00\x00'
>>> ^Z

C:\Users\Bruno>py -3.5
Python 3.5.1 (v3.5.1:37a07cee5969, Dec  6 2015, 01:54:25) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import struct
>>> struct.pack(">I", 0x80000000)
b'\x80\x00\x00\x00'
>>>
```
The result is not same type, you should keep the lambda definition.

**Comment by roks0n** [comment]
Ty @Moustikitos, I've changed it now. I'll try to work more towards increasing overall test coverage in my upcoming PRs.

**Comment by Moustikitos** [comment]
Nice work !

**Pull request merged**